app/utils/feature-gate/types.ts

```diff
@@ -1,10 +1,16 @@
 export type FeatureInfoType = {
-    key: string;
+    key: string | null;
```

Comment:
Is it really possible that key will be `null`? That will likely break https://github.com/solana-foundation/explorer/pull/578/files#diff-6451b87f057460f80ba5f4b9500d4a157eff6dabf260e294c0a69d62b04f9c1bR260

Reply:
#578